src/main/java/com/ProjectEureka/backend/models/Question.java

```diff
@@ -0,0 +1,53 @@
+package com.ProjectEureka.backend.models;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
```

Comment:
Remove Lombok.getter and lombok.setter as they are already covered in the lombok.data

Reply:
done in the latest commit